### DISCUSSION TITLE
Runtime error for Netlogo

### DISCUSSION THREAD

**Discussion by Anniee1177** [pybricks/support#645] [Q&A] [answered]
Hi, i am doing a geographic model based on Netlogo combined with arcgis and Qgis. But having problem loading data with load-patch-data command:
![image](https://user-images.githubusercontent.com/98409741/164866715-96c2e9d1-828c-4d3f-9798-8c8d0fceda04.png)
Looking forward to your help
Thanks

**Comment by laurensvalk** [comment, marked as answer]
Are you sure this is the right repository?

Pybricks is firmware that makes battery powered LEGO hubs programmable.
